### DISCUSSION TITLE
Storing shell history?

### DISCUSSION THREAD

**Discussion by dabeeeenster** [twpayne/chezmoi#2313] [Q&A] [answered]
Love this tool - thanks so much. 

I'm using https://github.com/ellie/atuin to store my shell history. I want to sync this with chezmoi. The problem is that the file changes on every command entered, so it's in a permanent state of being out of sync with the working repo. Are there any neat solutions to this?  

Or is this just a generally bad idea?!

**Comment by twpayne** [comment, marked as answer]
chezmoi currently doesn't have a good way to share/store shell history. chezmoi's only shared state between machines is your git repo, and creating a commit every time a command is entered would quickly become cumbersome.

github.com/ellie/atuin looks like a great solution. Maybe the best approach would be to use atuin but use chezmoi to install and configure atuin on your machines?
